config/crd/bases/infrastructure.cluster.x-k8s.io_openstackclusters.yaml

```
@@ -5534,8 +5535,12 @@ spec:
                     type: string
                 type: object
               subnets:
-                description: If NodeCIDR cannot be set this can be used to detect
-                  existing IPv4 and/or IPv6 subnets.
+                description: |-
+                  Subnets specifies existing subnets to use if not ManagedSubnets are
+                  specified. All subnets must be in the network specifed by Network.
```

Just to make the lint happy :)
```suggestion
                  specified. All subnets must be in the network specified by Network.
```

---

Thank you! Committed and squashed.